tutorials/sdf_to_usd_arms.md

```diff
@@ -0,0 +1,97 @@
+# Convert a SDF to a USD
```

Comment:
Can the tutorial be renamed to something more specific, since it covers more than just converting SDF to USD? Perhaps it can be renamed to `Manipulating SDF Panda Robot Arm in Isaac Sim with USD`